homeassistant/components/bluesound/media_player.py

```diff
@@ -70,21 +67,6 @@
 UPDATE_PRESETS_INTERVAL = timedelta(minutes=30)
 UPDATE_SERVICES_INTERVAL = timedelta(minutes=30)
 
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
```

Comment:
We should retain the schema and use an import flow to import the YAML and create an issue to indicate the user that they should remove the YAML.

For an example checkout "incomfort"